bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -134,6 +135,28 @@ where
 	No,
 }
 
+pub struct EnsureOriginWithControlFlag<T, F, A>(core::marker::PhantomData<(T, F, A)>);
```

Comment:
not sure we need this somewhat complicated struct..

Instead of switching the boolean generic param on this struct, we can start with simple `EnsureRoot` struct, then after audit switch (in runtime config) to simple `EnsureSigned` struct.
(hopefully somewhere in SDK there are already definitions for above simple structs and you don't even need to redefine them)